mysql-test/suite/rocksdb/t/deadlock_tracking.test

```diff
@@ -0,0 +1,47 @@
+set @prior_lock_wait_timeout = @@rocksdb_lock_wait_timeout; 
+set @prior_deadlock_detect = @@rocksdb_deadlock_detect; 
+set global rocksdb_deadlock_detect = on; 
+set global rocksdb_lock_wait_timeout = 10000;
+
+let $engine=rocksdb;
+
+let $prior_set_lwt = `select concat('set @prior_lock_wait_timeout = @@',
```

Comment:
Also the global vars need to be reset to their original values before the test. The test suite throws an error otherwise